package.json

```diff
@@ -32,47 +32,47 @@
   },
   "homepage": "https://github.com/tjoskar/ng-lazyload-image#readme",
   "peerDependencies": {
-    "@angular/core": ">=2.0.0",
-    "rxjs": ">=5.0.0"
+    "@angular/core": ">=6.0.0",
+    "rxjs": ">=6.0.0"
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "^4.2.6",
-    "@angular/common": "^4.2.6",
-    "@angular/compiler": "^4.2.6",
-    "@angular/compiler-cli": "^4.2.6",
-    "@angular/core": "^4.2.6",
-    "@angular/platform-browser": "^4.2.6",
-    "@angular/platform-browser-dynamic": "^4.2.6",
-    "@angular/platform-server": "^4.2.6",
-    "@angular/router": "^4.2.6",
+    "@angular/animations": "^6.0.0-rc.1",
+    "@angular/common": "^6.0.0-rc.1",
+    "@angular/compiler": "^6.0.0-rc.1",
+    "@angular/compiler-cli": "^6.0.0-rc.1",
+    "@angular/core": "^6.0.0-rc.1",
+    "@angular/platform-browser": "^6.0.0-rc.1",
+    "@angular/platform-browser-dynamic": "^6.0.0-rc.1",
+    "@angular/platform-server": "^6.0.0-rc.1",
+    "@angular/router": "^6.0.0-rc.1",
     "@types/core-js": "^0.9.46",
     "@types/mocha": "^5.0.0",
-    "@types/node": "^9.4.0",
-    "awesome-typescript-loader": "^3.4.1",
+    "@types/node": "^9.6.2",
+    "awesome-typescript-loader": "^5.0.0-1",
     "copyfiles": "^2.0.0",
     "core-js": "^2.5.3",
-    "electron": "^1.7.12",
+    "electron": "^2.0.0-beta.7",
```

Comment:
Why upgrading to a beta version?

Reply:
oops. should be fixed now.

I ran a script to update all the angular packages to latest RC (since there's so many it's a pain to do by hand) & I forgot to exclude non-angular packages.